jni/src/faiss_wrapper.cpp

```diff
@@ -223,6 +226,76 @@ void knn_jni::faiss_wrapper::CreateIndexFromTemplate(knn_jni::JNIUtilInterface *
     faiss::write_index(&idMap, indexPathCpp.c_str());
 }
 
+void knn_jni::faiss_wrapper::CreateBinaryIndexFromTemplate(knn_jni::JNIUtilInterface * jniUtil, JNIEnv * env, jintArray idsJ,
```

Comment:
This class is not right place to have this , we already have BinaryIndexService can we have all Binary related functions to that BinaryIndexService.

Reply:
Yes, we will add it in BinaryIndexService by refactoring.